pkg/rtc/mediatrackreceiver.go

```diff
@@ -206,6 +207,14 @@ func (t *MediaTrackReceiver) SetupReceiver(receiver sfu.TrackReceiver, priority
 }
 
 func (t *MediaTrackReceiver) SetPotentialCodecs(codecs []webrtc.RTPCodecParameters, headers []webrtc.RTPHeaderExtensionParameter) {
+	// The potential codecs have not published yet, so we can't get the actual Extensions, assume they will have
+	// same extensions as the primary codec except for the dependency descriptor
```

Comment:
is this assumption going to affect functionality in a bad way? Guessing not, just making sure. If the assumption is not going to affect functionality, may add a comment line to say that it is okay to assume because of XYZ?

Reply:
Also, if we use DD with non-SVC at some point, guessing this needs to be removed. Maybe, we will never use it, but would be good to add that also as a comment?

Reply:
It based on the browser uses same extensions for all video codecs and we only add DD extension for svc codec, so it will work well. Append the comment to it.